src/test/java/eu/yals/test/utils/elements/VaadinElement.java

```diff
@@ -9,58 +9,58 @@
+    private final TestBenchElement testBenchElement;
 
-  public static VaadinElement wrap(TestBenchElement element) {
-    return new VaadinElement(element);
-  }
-
-  public VaadinElement(TestBenchElement element) {
-    super(element);
-    this.testBenchElement = element;
-  }
+    public static VaadinElement wrap(final TestBenchElement element) {
+        return new VaadinElement(element);
+    }
 
-  public void inputShouldBeEmpty() {
-    Assert.assertNotNull("No such element found", testBenchElement);
-    Assert.assertTrue("Input is not TextField", testBenchElement instanceof TextFieldElement);
+    public VaadinElement(final TestBenchElement element) {
+        super(element);
+        this.testBenchElement = element;
+    }
 
-    TextFieldElement element = (TextFieldElement) testBenchElement;
-    Assert.assertTrue("Input is not empty", StringUtils.isBlank(element.getValue()));
-  }
+    public void inputShouldBeEmpty() {
+        Assert.assertNotNull("No such element found", testBenchElement);
+        Assert.assertTrue("Input is not TextField", testBenchElement instanceof TextFieldElement);
 
-  public void errorTextHas(String phrase) {
-    Assert.assertNotNull("No such element found", testBenchElement);
-    Assert.assertTrue(
-        "Element is not NotificationElement", testBenchElement instanceof NotificationElement);
+        TextFieldElement element = (TextFieldElement) testBenchElement;
+        Assert.assertTrue("Input is not empty", StringUtils.isBlank(element.getValue()));
+    }
 
-    NotificationElement notification = (NotificationElement) testBenchElement;
-    String notificationText = getNotificationText(notification);
-    Assert.assertTrue(
-        String.format("Error text: %s does not contain phrase: %s", notificationText, phrase),
-        StringUtils.containsIgnoreCase(notificationText, phrase));
-  }
+    public void errorTextHas(String phrase) {
+        Assert.assertNotNull("No such element found", testBenchElement);
+        Assert.assertTrue(
+                "Element is not NotificationElement", testBenchElement instanceof NotificationElement);
 
-  private String getNotificationText(NotificationElement notification) {
-    final String BUTTON_TEXT = "OK";
-    String notificationText = notification.getText();
-    if (!notificationText.equalsIgnoreCase(BUTTON_TEXT)) {
-      return notificationText;
+        NotificationElement notification = (NotificationElement) testBenchElement;
+        String notificationText = getNotificationText(notification);
+        Assert.assertTrue(
+                String.format("Error text: %s does not contain phrase: %s", notificationText, phrase),
+                StringUtils.containsIgnoreCase(notificationText, phrase));
     }
 
-    log.info(
-        "Seems like notification.getText() method gave button's text. Getting text using JS methods instead");
+    private String getNotificationText(NotificationElement notification) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter notification should be final.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)